README.md

```diff
@@ -5,9 +5,9 @@ Small repo done in cab ride
 A simple repo used to track progress of codechef monthly contests from JAN15-JUNE17 (Can be increased or decresed by modifying code)
 
 ## What to do?
-* Install virtual env
+* Install virtual env 
 * Create an environment using virtual env `virtualenv envName -p python3`
-* Activate the environment and install the requirements `source bin/activate; pip3 install -r req.txt`
+* Activate the environment and install the requirements `source envName/bin/activate; pip3 install -r req.txt`
```

Comment:
see the thing is when the user creates a virtual environment ( for example the name of the virtual env be "contest" ) then the system gathers all then environment variables and all together in a folder named contest in the project ContestDone.
now what i have done is activating the virtual environment from the main django project i.e. ContestDone folder.
And what you are saying is to first do cd contest and go inside the folder and then activate the virtual environment.
Both options are welcome but in your option, after we have activated the virtual env from the contest folder we again have to do cd .. and come out of the folder because manage.py and helper.py are not present in the contest folder rather they are there in the ContestDone folder (which is the django project) . So we will again have to add a step of cd .. to come out of the virtual env folder if we follow your option. And moving manage.py and helper.py to the virtual env folder is not possible as there is a link between .py files and it may break due to unnecessary movement of files.
And in the other option no such things is needed to be done as we are specifying the name of the virtual env while activating it.

your opinion?